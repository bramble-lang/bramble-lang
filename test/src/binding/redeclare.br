fn my_main() {
    x: i32 := 5;
    x: i32 := 10;
    return;
}