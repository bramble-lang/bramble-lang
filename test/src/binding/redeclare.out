Error: Semantic: L3: x already declared
